src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -59,6 +59,7 @@ const defaultProps = {
 
 function MoneyRequestConfirmPage(props) {
     const {windowHeight} = useWindowDimensions();
+    const isDistanceRequest = props.selectedTab === CONST.TAB.DISTANCE;
```

Comment:
I think there is a refactor on the URLs related to money requests so that we can embed the info on the kind of money requests being made inside the URL so we don't have to this kind of awkward conditional checks